app/controllers/responsiblepeople/PersonNonUKPassportController.scala

```diff
@@ -85,4 +69,15 @@ class PersonNonUKPassportController @Inject()(
         }
   }
 
+  private def redirectToNextPage(result: Option[CacheMap], index: Int,
+                                 edit: Boolean, flow: Option[String] )(implicit request: Request[AnyContent]) = {
```

Comment:
please put first group of parameters in line 72 and implicit group in 73

Reply:
Done